app/actors/hyrax/actors/paged_resource_actor.rb

```diff
@@ -3,6 +3,19 @@
 module Hyrax
   module Actors
     class PagedResourceActor < Hyrax::Actors::BaseActor
+      def create(env)
```

Comment:
You could break this up into two commits where you (1) copy in the inherited `#create` method from `Hyrax::Actors::BaseActor`, and then (2) add in the new line for `apply_ocr_option...`, if you want to follow that pattern for making changes to stock Hyrax behavior more cleanly isolated.

Reply:
Good idea, I split commits as suggested.  I tried to `super` the method to avoid that, but there was just something about the atomic nature of the save when broken up that just wouldn't work so I was forced to do the steps explicitly and in order before save.